tests/Performance/runPerformanceTests.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+require __DIR__.'/../../alt_autoload.php-dist';
+
+// FIXME: Autoload these.
+require_once __DIR__.'/Exception/PerformanceFailException.php';
+require_once __DIR__.'/Test/AbstractPerformanceTest.php';
+require_once __DIR__.'/Test/DocumentDictionaryCacheTest.php';
```

Comment:
Autoloading for test related classes is already possible (see [this](https://github.com/smalot/pdfparser/blob/master/composer.json#L27-L31)). Please use `vendor/autoload.php` instead.

```suggestion
require __DIR__.'/../../vendor/autoload.php';
```

Reply:
You need to run `composer update` before you start your tests.